packages/material-ui-utils/src/getDisplayName.js

```diff
@@ -50,6 +50,8 @@ export default function getDisplayName(Component) {
     switch (Component.$$typeof) {
```

Comment:
I wonder, could `isFragment` and `isMemo` have been used instead?

Reply:
I tried switching to `typeOf(Component)`, however IIRC it appeared that for memo `typeOf` actually returns `Component.type.$$typeof`. Not sure isMemo would work since most of the `is*` functions use `typeOf`.

Reply:
Right, so it's already as good as it can get.

Reply:
Yes, for now.

My real end-goal is for React to export `getComponentName`, in which case we won't need to bother with these hacks at all (see [15207](https://github.com/facebook/react/issues/15207#issuecomment-587187778) and [14319](https://github.com/facebook/react/issues/14319)).